pkg/controller/controller_binding_test.go

```diff
@@ -2444,7 +2444,8 @@ func TestReconcileBindingWithOrphanMitigationReconciliationRetryTimeOut(t *testi
 			SecretName:         testServiceBindingSecretName,
 		},
 	}
-	startTime := metav1.NewTime(time.Now().Add(-7 * 24 * time.Hour))
+	startTime := metav1.NewTime(time.Now().Add(-1 * time.Hour))
```

Comment:
This test was meant to check the reconciler's reaction to a binding currently undergoing orphan mitigation whose retry duration has been exceeded. We should have a different test that checks this situation.

Reply:
It wasn't clear to me what the test was actually testing. Because the binding did not have a Current person set, it wasn't actually testing the scenario of reconciliation timing out. The OperationStartTime was being set to Now by the controller. I tried to deduce by the check that the reconcile function was expected to return an error that the test was supposed to test what happened when the result of the deprovision request was a timeout error.

Reply:
Ah, okay. Looks like the intent of the test was to test the logic where an error during an orphan mitigation unbind when the retry duration has been exceeded gets us into the logic [here](https://github.com/staebler/service-catalog/blob/18b655c6cbe9d33b7792767792f3c23015953435/pkg/controller/controller_binding.go#L735). However, the test isn't actually checking for that; it's just verifying the normal orphan mitigation reconciliation error state, and that's why fixing the `CurrentOperation` bug made it actually start to fail, and setting the `OperationStartTime` to something sooner made it start to pass.

I'm fine with putting a bandaid on it for now and fixing it later.